Gosu/Window.h

```diff
@@ -31,11 +47,11 @@ Gosu_Window* Gosu_Window_create(int width, int height, int fullscreen, double up
 // callbacks
 void Gosu_Window_update(Gosu_Window* window, void function());
 void Gosu_Window_draw(Gosu_Window* window, void function());
-void Gosu_Window_button_down(Gosu_Window* window, void function());
-void Gosu_Window_button_up(Gosu_Window* window, void function());
-void Gosu_Window_drop(Gosu_Window* window, void function());
-void Gosu_Window_needs_redraw(Gosu_Window* window, void function());
-void Gosu_Window_needs_cursor(Gosu_Window* window, void function());
+void Gosu_Window_button_down(Gosu_Window* window, void function(unsigned btn));
```

Comment:
Since these are effectively setters, how about renaming them to `Gosu_Window_set_button_down` or `Gosu_Window_on_button_down` etc.?